spec/integration/thousand_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'open3'
```

Comment:
You can't include standard library into our test suite, has to be done in child processes. The reason behind this is it would allows us to accidentally depend on this functionality and cause false positives in peoples code.

Reply:
There is an `in_sub_process` helper for this.